composer.json

```diff
@@ -4,9 +4,9 @@
     "type": "symfony-bundle",
     "license": "MIT",
     "require": {
-        "php": "8.*",
+        "php": ">=8.1",
```

Comment:
Jestem przeciwny, nie ma żadnej gwarancji że paczka będzie działać z PHP 9.0, albo 10.0, albo 15.0 a dokładnie to teraz jest obiecane